tachyon/base/openmp_util.h

```diff
@@ -12,9 +12,11 @@
 #define OPENMP_PARALLEL_FOR(expr) _Pragma("omp parallel for") for (expr)
 #define OPENMP_PARALLEL_NESTED_FOR(expr) \
   _Pragma("omp parallel for collapse(2)") for (expr)
+#define OPENMP_FOR(expr) _Pragma("omp for") for (expr)
```

Comment:
Originally, I implemented parallelization in prover/verifier such that OPENMP would be opened only one time for the multiple parallel for loops I had, but after I realized that I only needed one parallel for, I removed the use of `OPENMP_FOR` from prover/verifier, yet kept the definition of `OPENMP_FOR`, since I know it'll need to be used in the future.